From 6ebcae4340dd19c556c7de890e90b3bb6cebe97d Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 19 Jun 2018 14:59:00 -0700
Subject: [PATCH] TRITON-527 cloudapi GetMachine should document
 'credentials=true' param Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
 Approved by: Marsell Kukuljevic <marsell@joyent.com>

---
 docs/index.md | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/docs/index.md b/docs/index.md
index 83b3756..2066f55 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -4327,7 +4327,9 @@ been purged from Triton.
 
 ### Inputs
 
-* None
+**Field**   | **Type** | **Description**
+----------- | -------- | ---------------
+credentials | Boolean  | Whether to include the generated credentials for instances, if present. Defaults to false.
 
 ### Returns
 
-- 
2.21.0

